### PR TITLE
TIP-706-enabled: Filter and Normalization

### PR DESCRIPTION
**Pull request by anaelChardan**
- [x] Review the PQB filter integration tests
- [x] Add index mapping integration tests 
- [x] Write the filter & Spec
- [x] Update the documentation in `search_specification.rst`
- [x] Update the mapping
- [x] Add the normalizer
- [x] Is the PIM installable ?

**Definition Of Done (for Core Developer only)**

| Q                                 | A
| --------------------------------- | ---
| Added Specs                       | Todo
| Added Behats                      | Todo
| Added integration tests           | Todo
| Changelog updated                 | Todo
| Review and 2 GTM                  | Todo
| Micro Demo to the PO (Story only) | Todo
| Migration script                  | -
| Tech Doc                          | -

`Todo`: Pending / Work in progress
`OK`: Done / Validated
`-`: Not needed

### CODE REVIEW

File: src/Pim/Bundle/CatalogBundle/Elasticsearch/Filter/StatusFilter.php [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+namespace Pim\Bundle\CatalogBundle\Elasticsearch\Filter;
+
+use Akeneo\Component\StorageUtils\Exception\InvalidPropertyTypeException;
+use Pim\Component\Catalog\Exception\InvalidOperatorException;
+use Pim\Component\Catalog\Query\Filter\FieldFilterInterface;
+use Pim\Component\Catalog\Query\Filter\Operators;
+
+/**
+ * Product status filter.
```
Comment (SamirBoulil) [inline src/Pim/Bundle/CatalogBundle/Elasticsearch/Filter/StatusFilter.php:11]: ... for an Elasticsearch query

**Review by SamirBoulil — APPROVED** [review]
GTM once CI is green

**Review by skeleton — APPROVED** [review]

File: src/Pim/Bundle/CatalogBundle/Elasticsearch/Filter/search_specification.rst [hunk 2]
```diff
@@ -258,6 +258,53 @@ EMPTY
...
+
+Filtering
+~~~~~~~~~
+Operators
+.........
+Equals (=)
+~~~~~~~~~~
+
+.. code-block:: php
+
```
Comment (jjanvier) [inline src/Pim/Bundle/CatalogBundle/Elasticsearch/Filter/search_specification.rst:279]: query part?

File: src/Pim/Bundle/CatalogBundle/Resources/elasticsearch/index_configuration.yml [hunk 3]
```diff
@@ -5,6 +5,8 @@ mappings:
                 type: 'keyword'
             family:
                 type: 'keyword'
+            enabled:
```
Comment (jjanvier) [inline src/Pim/Bundle/CatalogBundle/Resources/elasticsearch/index_configuration.yml:8]: why do we need that for a boolean?

File: src/Pim/Bundle/CatalogBundle/Elasticsearch/Filter/search_specification.rst (hunk 2 above)
Comment (anaelChardan) [reply src/Pim/Bundle/CatalogBundle/Elasticsearch/Filter/search_specification.rst:279]: Node useful when it concerns only one bool clause :)

File: src/Pim/Bundle/CatalogBundle/Resources/elasticsearch/index_configuration.yml (hunk 3 above)
Comment (anaelChardan) [reply src/Pim/Bundle/CatalogBundle/Resources/elasticsearch/index_configuration.yml:8]: Yes, with @SamirBoulil we think that for fields it's better to be exhaustive

**Pull request merged**